### ISSUE TITLE
Aggressive Friday scrape did not capture change in event items

### ISSUE THREAD

**Issue by hancush** [datamade/scrapers-us-municipal#44]
[The Regular Board Meeting](http://webapi.legistar.com/v1/metro/events/1638) had a change in event items, however that change was not captured by our aggressive Friday night full scrapes. 

I pulled two log files covering `02/21/2020 01:45:08` to `02/23/2020 01:27:23`. (For posterity, I unzipped the archived log files like: `gunzip /tmp/lametro.log.X -k -c > lametro.log.X` where X is some log index.)

Consulting those logs, the last time we have a record of the board meeting being scraped prior to the issue is around 6:45 Friday night.

```
ubuntu@ip-10-0-0-80:~$ grep 1638 lametro.log.3
02/21/2020 15:00:32 INFO scrapelib: GET - https://metro.legistar.com/MeetingDetail.aspx?LEGID=1638&GID=557&G=A5FAA737-A54D-4A6C-B1E8-FF70F765FA94
02/21/2020 15:01:34 INFO scrapelib: GET - http://webapi.legistar.com/v1/metro/events/1638/eventitems
02/21/2020 16:00:32 INFO scrapelib: GET - https://metro.legistar.com/MeetingDetail.aspx?LEGID=1638&GID=557&G=A5FAA737-A54D-4A6C-B1E8-FF70F765FA94
02/21/2020 16:01:35 INFO scrapelib: GET - http://webapi.legistar.com/v1/metro/events/1638/eventitems
02/21/2020 16:26:24 INFO pupa: save bill 2017-0496 in 2017 as bill_31638240-54f9-11ea-b728-122a3d729da3.json
02/21/2020 17:00:32 INFO scrapelib: GET - https://metro.legistar.com/MeetingDetail.aspx?LEGID=1638&GID=557&G=A5FAA737-A54D-4A6C-B1E8-FF70F765FA94
02/21/2020 17:01:31 INFO scrapelib: GET - http://webapi.legistar.com/v1/metro/events/1638/eventitems
02/21/2020 18:00:32 INFO scrapelib: GET - https://metro.legistar.com/MeetingDetail.aspx?LEGID=1638&GID=557&G=A5FAA737-A54D-4A6C-B1E8-FF70F765FA94
02/21/2020 18:01:32 INFO scrapelib: GET - http://webapi.legistar.com/v1/metro/events/1638/eventitems
02/21/2020 18:33:30 INFO scrapelib: GET - https://metro.legistar.com/MeetingDetail.aspx?LEGID=1638&GID=557&G=A5FAA737-A54D-4A6C-B1E8-FF70F765FA94
******* 02/21/2020 18:44:01 INFO scrapelib: GET - http://webapi.legistar.com/v1/metro/events/1638/eventitems *******
02/21/2020 20:07:32 INFO pupa: save vote_event 2015 - 2016-01-20 - RECOMMENDED FOR APPROVAL as vote_event_16382c72-5518-11ea-b6d3-122a3d729da3.json
```

We don't have another log entry for it being scraped again for nearly 24 hours.

```
ubuntu@ip-10-0-0-80:~$ grep 1638 lametro.log.2
02/22/2020 11:45:08 INFO pupa: save post Los Angeles County Board Supervisor, District 3 as post_11638948-559b-11ea-be55-122a3d729da3.json
02/22/2020 16:30:01 INFO pupa: save post Los Angeles County Board Supervisor, District 1 as post_dda71638-55c2-11ea-ba24-122a3d729da3.json
02/22/2020 18:26:07 INFO scrapelib: GET - https://metro.legistar.com/MeetingDetail.aspx?LEGID=1638&GID=557&G=A5FAA737-A54D-4A6C-B1E8-FF70F765FA94
******* 02/22/2020 18:36:51 INFO scrapelib: GET - http://webapi.legistar.com/v1/metro/events/1638/eventitems *******
```

What's up with that???

**Comment by hancush** [comment]
Meanwhile, neither the aggressive full scraping nor the nightly full scrape captured changes to this event on the upgrade site. When I ran the event scrape by hand with a window of 7, it was captured.

**Comment by hancush** [comment]
This was resolved by #102 and https://github.com/datamade/la-metro-councilmatic/issues/557#issuecomment-604039556.
